contracts/StablecoinConverter.sol

```diff
@@ -102,71 +143,32 @@ contract StablecoinConverter is EpochTokenLocker {
         return orders[msg.sender].length - 1;
     }
 
-    function cancelOrder(
-        uint id
-    ) public {
+    /** @dev a user facing function used to cancel orders (sets order expiry to previous batchId)
+      * @param id referencing the index of user's order to be canceled
+      */
+    function cancelOrder(uint id) public {
         orders[msg.sender][id].validUntil = getCurrentBatchId() - 1;
         emit OrderCancelation(msg.sender, id);
     }
 
-    function freeStorageOfOrder(
-        uint[] memory id
-    ) public {
-        for (uint i = 0; i < id.length; i++) {
-            require(orders[msg.sender][id[i]].validUntil + 1 < getCurrentBatchId(), "Order is still valid");
-            delete orders[msg.sender][id[i]];
-        }
-    }
-
-    function tokenAddressToIdMap(address addr) public view returns (uint16) {
-        return IdToAddressBiMap.getId(registeredTokens, addr);
-    }
-
-    function tokenIdToAddressMap(uint16 id) public view returns (address) {
-        return IdToAddressBiMap.getAddressAt(registeredTokens, id);
-    }
-
-    function getEncodedAuctionElements() public view returns (bytes memory elements) {
-        if (allUsers.size() > 0) {
-            address user = allUsers.first();
-            bool stop = false;
-            while (!stop) {
-                for (uint i = 0; i < orders[user].length; i++) {
-                    Order memory order = orders[user][i];
-                    elements = elements.concat(encodeAuctionElement(
-                        user,
-                        getBalance(user, tokenIdToAddressMap(order.sellToken)),
-                        order
-                    ));
-                }
-                if (user == allUsers.last) {
-                    stop = true;
-                } else {
-                    user = allUsers.next(user);
-                }
-            }
+    /** @dev a user facing function used to delete expired orders
```

Comment:
maybe mention that this is worth doing for the gas refund (and assumes that orders are already invalid)